transforms/universal/ededup/python/src/ededup_transform_base.py

```diff
@@ -73,26 +123,32 @@ def __init__(self, config: dict):
         """
         super().__init__(config)
         self.doc_column = config.get("doc_column", "contents")
+        self.doc_id_column = config.get("doc_id_column", "document_id")
```

Comment:
again, please use a constant for these key values both here and in the parser setup.

Reply:
Its going to be an import nightmare

Reply:
no one is required to use the constants, but it would be highly recommended.  Mostly I suggest it for your own implementation internally.  Makes for better error checking and less delicate code.

Reply:
done